scanpipe/models.py

```diff
@@ -432,12 +432,16 @@ def add_error(self, error, model, details=None):
         Create a ProjectError record from the provided `error` Exception for this
         project.
         """
+        traceback = ""
```

Comment:
Have you tried if the simpler `traceback.format_exc` may work, e.g. if you are already in an exception?